### PR TITLE
Treasury deposit and tidy

### PR DESCRIPTION
**Pull request by bdmason**
- added treasury deposit event
- removed generated files (see makefile, postinstall hook, & typegen folder)
- added prettier (with husky to force formats on commit)
- added support for litmus, litentry & khala

### CODE REVIEW

**Review by jdomingos — APPROVED** [review]
No issues detected and I read the code :D

File: .env [hunk 1]
```diff
@@ -3,3 +3,4 @@ DB_PASS=squid
 DB_PORT=23798
 PROCESSOR_PROMETHEUS_PORT=3000
 GQL_PORT=4350
+NETWORK=litmus
```
Comment (markdavison) [inline .env:6]: I don't think this wants committing :)

File: package.json [hunk 2]
```diff
@@ -8,26 +8,32 @@
     "build": "rm -rf lib && tsc",
     "db:migrate": "npx squid-typeorm-migration apply",
     "processor:start": "node lib/processor.js",
```
Comment (markdavison) [inline package.json:10]: I think we can add some dev scripts in here to make swapping networks easy.

File: .env (hunk 1 above)
Comment (bdmason) [reply .env:6]: It is ignored in production and only used locally, agree with your comment below though. I'll do it in another PR, I'm thinking of a script that gets all networks by reading the typegen folder then asks you to choose when running `npm start`

File: package.json (hunk 2 above)
Comment (bdmason) [reply package.json:10]: See above, I'll do it in a new PR as it will want a closer review of its own

**Pull request merged**